docs/what.rst

```diff
@@ -0,0 +1,29 @@
+=====================
+What is a Certificate?
+=====================
+
+A public key or digital *certificate* (formerly called an SSL certificate) uses a public key 
+and a private key to enable secure communication between a client program (web browser, email client, 
+etc.) and a server over an encrypted SSL (secure socket layer) or TLS (transport layer security) connection.
+The certificate is used both to encrypt the communication and to identify the server. The certificate
+includes information about the key, information about the server identity, and the digital signature
+of the certificate issuer. If the issuer is trusted by the software that initiates the communication,
+and the signature is valid, then the key can be used to communicate securely with the server identified by 
+the certificate. Using a certificate is a good way to prevent "man-in-the-middle" attacks, in which
+someone in between you and the server you think you are talking to is able to insert their own (harmful)
+content.
+
+You can use Certbot to easily obtain and configure a free certificate from Let's Encrypt, a
+joint project of EFF, Mozilla, and many other sponsors.
+
+Certificates and Lineages
+=========================
+
+Certbot introduces the concept of a *lineage,* which is a collection of all the versions of a certificate
+plus Certbot configuration information maintained for that certificate from
+renewal to renewal. Whenever you renew a certificate, Certbot keeps the same configuration unless
+you explicitly change it. If you add domains, you can either add them to an existing lineage or create
+a new one. 
+
+See also:
+:ref:`updating_certs`
```

Comment:
@schoen suggests;

What is a Certificate?

A digital certificate (historically called an SSL certificate) enables secure communication between a client program (web browser, email client, etc.) and a server over an encrypted TLS (Transport Layer Security) connection, such as HTTPS for web connections. (TLS was previously known as SSL.)

Browsers and other software commonly require a certificate when making a secure connection to an Internet service. You can use Certbot to easily obtain and configure a free certificate from Let's Encrypt, a joint project of EFF, Mozilla, and many other sponsors. These certificates are accepted by mainstream browsers and other software.

The certificate contains a public key. The certificate's recipient—the operator of the site or service described by the certificate—also has the corresponding private key, which needs to be kept secret. The certificate is used to identify the server when establishing a secure connection. The certificate includes information about the public key, information about the server identity, and the digital signature of the certificate authority that issued the certificate. If the issuer is trusted by the software that initiates the communication, and the signature is valid, then the certificate confirms that the public key can be used to communicate securely with the server identified by the certificate. The main security benefit of using a certificate is the prevention of "man-in-the-middle" attacks, in which someone in between you and the server you think you are talking to is able to spy on you or insert their own (harmful) content.

Once a certificate has been obtained, it also needs to be installed in whatever server software is going to use it. This is generally done by editing that software's configuration files. Certbot can take care of this step for Apache and Nginx web servers.

Certificates and Lineages

Certificates eventually expire (in Let's Encrypt's case, every ninety days). "Renewing" a certificate really means getting a fresh certificate for the same domain names, which can be installed in place of the old one. In most configurations, Certbot can do this automatically using `certbot renew`.

Certbot introduces the concept of a lineage, which is a collection of all the versions of a certificate and its associated files from renewal to renewal, plus Certbot configuration information for that certificate. Whenever you renew a certificate, Certbot keeps the same configuration unless you explicitly change it. If you add domains, you can either add them to an existing lineage or create a new one.

You can view a list of lineages, and their names, by running `certbot certificates`; for most operations with Certbot, you can specify which lineage you want to act on using `--cert-name`.

See also: :ref:`updating_certs`